stack.js

```diff
@@ -95,6 +95,11 @@ class Stack {
     }
   }
 
+  list() {
+    this.resolve();
```

Comment:
An offtopic thought: `resolve` should probably return the execution order

Reply:
Yep, it really should return the execution order. Can you refactor this?

Reply:
Yes, will do